python/desc/imsim/imSim.py

```diff
@@ -178,3 +202,32 @@ def extract_objects(df):
     phosim_galaxies = \
         phosim_galaxies.query("halfLightSemiMajor >= halfLightSemiMinor")
```

Comment:
That's fair enough.  I'll add a function to make these checks and report any issues if they occur.